package.json

```diff
@@ -71,6 +71,6 @@
     "style-loader": "0.13.1"
   },
   "dependencies": {
-    "@bufferapp/components": "0.1.16"
+    "@bufferapp/components": "0.1.17"
```

Comment:
Hey, @alvarezmelissa87! I wonder how it feels for us to bump up the version number on `master` or perhaps even a new branch? 😄 I keep hitting up the number myself within branches, too!